packages/web-twig/src/Resources/components/FieldGroup/FieldGroup.twig

```diff
@@ -1,5 +1,6 @@
 {# API #}
 {%- set props = props | default([]) -%}
+{%- set _children = block('content') is defined ? block('content') : null -%}
```

Comment:
This conditions is necessary for snapshot test.
But it's discutable if we need to check empty `FieldGroup`…

Reply:
Ok, this is not necessary and I have removed this entirely.